client/components/sub-masterbar-nav/navbar.jsx

```diff
@@ -82,16 +85,20 @@ export default class Navbar extends Component {
 	}
 
 	toggleList = () => {
-		this.setState( ( state ) => ( {
-			collapsed: ! state.collapsed
+		this.setListCollapsed( ! this.state.collapsed );
+	}
+
+	setListCollapsed( collapsed ) {
+		this.setState( () => ( {
+			collapsed: collapsed
```

Comment:
I think React 16 will default to this? To make it clear that `setState` is async. Not sure :)